resources/translations/en.edn

```diff
@@ -139,6 +143,7 @@
                                   :decision-requested "Decision requested"
                                   :draft-saved "Modified"
                                   :member-added "Member added"
+                                  :member-invited "Invited member"
```

Comment:
This is quite splitting hair, but would prefer "Member invited" for consistency.

Reply:
Yes it's better like that. Matches the others.